app/lib/authentication/strategy/token.rb

```diff
@@ -1,9 +1,11 @@
 module Authentication
   module Strategy
 
-    class SSO < Base
+    class Token < Base
```

Comment:
This was what you mentioned that it SSO meant impersonation token. Maybe name it `ImpersonationToken` then? Or you envision other use cases?

Reply:
I considered renaming this and even started writing the changes, but this [test](https://github.com/3scale/porta/blob/2819305ef8d327caebfa169ef1f90be9ea162d15/test/integration/sessions_test.rb#L126-L138) made me change my mind. Apparently at some point we used this authentication from the developer portal. From the test, I deduce it's related to forums. So most probably not used anymore, but just in case, I think it's better to stick to `Token`